src/main/kotlin/io/github/gradlenexus/publishplugin/internal/NexusClient.kt

```diff
@@ -54,6 +54,12 @@ open class NexusClient(private val baseUrl: URI, username: String?, password: St
                                 .build())
                     }
         }
+        httpClientBuilder.addInterceptor { chain ->
+            val version = javaClass.`package`.implementationVersion
```

Comment:
What will be used when running from IDE? Maybe some safe fallback value `undefined` or `local` or `devel` could be explicitly used?

Reply:
That depends how you've set up your IDE. If test execution is delegated to Gradle, it works. If IDEA is used directly, it won't.

Reply:
I changed it so it works either way.

Reply:
Thanks. I just didn't want to get NPE or something if that property is not set.